CHANGES.md

```diff
@@ -1,5 +1,20 @@
 # Changes
 
+## edge-21.7.4
+
+This release continues to focus on dependency updates. It also adds proxy a
```

Comment:
nit: Fine to keep this, but I think from now one we're always going to have quite a few dependency updates so it's probably not worth mentioning in every edge.